.github/workflows/test.yml

```diff
@@ -35,6 +35,25 @@ jobs:
         uses: ./
       - run: uv sync
         working-directory: __tests__/fixtures/uv-project
+  test-default-version-cache: # https://github.com/astral-sh/setup-uv/issues/171
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, macos-14]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install latest version
+        uses: ./
+      - name: Re-run setup-uv
+        id: re-setup-uv
+        uses: ./
+      - name: Verify uv cache hit
+        run: |
+          if [ "$UV_CACHE_HIT" != "true" ]; then
+            exit 1
+          fi
+        env:
+          UV_CACHE_HIT: ${{ steps.re-setup-uv.outputs.uv-cache-hit }}
```

Comment:
@eifinger 
> Can you already provide more detail on how the new output can be used?

Currently, it's only being used to test whether the cache is working correctly. If you can confirm that the cache is functioning properly through other means, you can delete it.